lib/modules/uscore_v3.1.0/us_core_diagnosticreport_lab_sequence.rb

```diff
@@ -93,7 +94,7 @@ def validate_resource_item(resource, property, value)
         validate_search_reply(versioned_resource_class('DiagnosticReport'), reply, search_params)
       end
 
-      test 'Server returns expected results from DiagnosticReport search by patient+code' do
+      test 'Server returns expected results from DiagnosticReport search by patient' do
```

Comment:
I'm not seeing a requirement for DiagnosticReport search by just patient: https://www.hl7.org/fhir/us/core/capstmnts.html#diagnosticreport

Reply:
That's from 3.0. Here's the most recent version: http://hl7.org/fhir/us/core/CapabilityStatement-us-core-server.html

Reply:
That confused me, too, but that's the 3.0.0 version of the spec. It's required in 3.1.0:
http://hl7.org/fhir/us/core/CapabilityStatement-us-core-server.html#diagnosticreport
These urls are really confusing

Reply:
Oh geez.  They probably should scrub that directory of old versions because these are still being indexed and i just go by the URL and not the header I see on the top regarding versions.